src/anm/player.js

```diff
@@ -380,6 +380,7 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
                         player.state.happens = C.LOADING;
                         player.fire(C.S_CHANGE_STATE, C.LOADING);
                         player.fire(C.S_LOAD, result);
+                        player._updateAudioVolumes();
```

Comment:
In the best case I imagined `mute/unmute/volume` methods could be added from inside of `audio.js` to `Player.prototype` and `Controls.prototype`. Though nowadays audio is not so very separate from Player as before, and it could complicate things more than improve, so whatever :).